app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -236,10 +236,18 @@ class DefaultToolbarMenu(
             onItemTapped.invoke(ToolbarMenu.Item.AddToHomeScreen)
         }
 
+        val accountManager = context.components.backgroundServices.accountManager
+        val notSignedIn = accountManager.authenticatedAccount()
```

Comment:
Is this an async call? Do we need to ensure that this has finished before evaluating `notSignedIn`?

Reply:
Also, should this be `signedIn` instead?

Reply:
I think it probably makes more sense to call it `account` to not confuse this with a boolean value.